### DISCUSSION TITLE
New container images registry

### DISCUSSION THREAD

**Discussion by machine424** [helmfile/helmfile#211] [Questions] [answered]
Hello,

Are we still using [quay.io/roboll/helmfile](http://quay.io/roboll/helmfile) for docker images? I cannot find v0.145.0  (released 6 days ago). Am I missing something?

**Comment by yxxhero** [comment]
@mumoshu

**Comment by engson** [comment]
It seems to have moved to github Container registry: https://github.com/orgs/helmfile/packages?repo_name=helmfile
Doesn't look one tag per release yet, canary might just be latest build.

**Comment by yxxhero** [comment, marked as answer]
https://github.com/helmfile/helmfile/pkgs/container/helmfile
image issue fixed. Please use v0.145.2 tag. Thanks very much.

**Reply by machine424** [reply]
Thanks.

**Reply by yxxhero** [reply]
@machine424 Welcome to Create discussions or PR. Thanks very much.
